modules/gltf/src/glb/glb-builder.js

```diff
@@ -145,15 +128,10 @@ export default class GLBBuilder {
     return this.json.accessors.length - 1;
   }
 
-  // Adds a binary image. Builds glTF "JSON metadata" and saves buffer reference
-  // Buffer will be copied into BIN chunk during "pack"
-  // TODO - this method should be renamed addImage
-  addImage(bufferViewIndex, {mimeType, width, height}) {
-    // DEPRECATED: We have renamed the addImage method, this is backwars compatibility
-    if (ArrayBuffer.isView(bufferViewIndex)) {
-      this.log.warn('GLBBuilder.addImage(): renamed to GLBBuilder.addImageData()');
-      return this.addImage(...arguments);
-    }
+  // Add an image entry from a bufferViewIndex, obtained from calling addBufferView()
+  // The bufferViewIndex and image information will be added to the gltf images
+  // property and the index of that entry returned.
+  addImageEntry(bufferViewIndex, {mimeType, width, height}) {
```

Comment:
If we are making this change, I think we should just drop the gltf knowledge completely here and just add the `bufferView`.

```
this.json.images.push({bufferView: bufferViewIndex});
```

loaders.gl offers utilities to cheaply extract the type and size if needed before decoding.

Ultimately we should also move the bufferView handling into glTF builder and offer a simpler (more compact) mechanism here, but this is good for now.